core/src/test/java/nl/nn/adapterframework/configuration/AutoRegisteringAdapterManager.java

```diff
@@ -0,0 +1,16 @@
+package nl.nn.adapterframework.configuration;
+
+import nl.nn.adapterframework.core.Adapter;
+
+public class AutoRegisteringAdapterManager extends AdapterManager {
```

Comment:
AutoRegisteringAdapterManager = AutoConfiguringAdapterManager

Gaat het er hier om dat het een soort MinimumViableAdapterManager voor unit tests is? Dan mag dat wel wat duidelijker worden.